.github/workflows/ci.yml

```diff
@@ -135,3 +136,27 @@ jobs:
 
       # Check that the release automation works as expected
       - run: scripts/release/test.sh
+
+  # This check is allowed to fail, while Marker is still unstable. It's mostly
+  # a sanity check, to make sure we don't have any unintentional API breakage :)
+  semver-checks:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+          path: './base'
+      - uses: actions/checkout@v4
+        with:
```

Comment:
```
with:
    ref: ${{ github.head_ref }}
```
Sadly doesn't work. I'm assuming that just specifying the ref, is not enough, and I would also need to change the remote to the fork, where the branch is from? Anyways, just leaving that part out, does the trick :)